array_api_tests/dtype_helpers.py

```diff
@@ -235,6 +235,7 @@ class MinMax(NamedTuple):
 if not hasattr(xp, "asarray"):
     default_int = xp.int32
     default_float = xp.float32
+    default_complex = xp.complex64 if api_version > "2021.12" else None
```

Comment:
So `xp` here actually is a wrapped array module, meaning attributes will exist even if not present in the library... except as we haven't bumped to the post-`2022.12` spec yet (#187) so this will currently give us an attribute error when testing libraries that don't have `complex64`.

Let's do something like

```suggestion
    # TODO: when api_version > '20221.12', just assign to xp.complex64,
    # otherwise default to None. Need array-api spec to be bumped first (#187).
    try:
        default_complex = xp.complex64
    except AttributeError:
        default_complex = None
    default_complex = xp.complex64 if api_version > "2021.12" else None
```

Reply:
Sure!